### PR TITLE
Add CI job names

### PR DESCRIPTION
**Pull request by szepeviktor**
Move comments to `name:`

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 jobs:
-  ## PHPUNIT
   phpunit:
+    name: PHPUnit
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-version: [8.1, 8.2, 8.3]
-        composer-flags: [null]
+        php-version: ['8.1', '8.2', '8.3']
```
Comment (szepeviktor) [inline .github/workflows/test.yaml:9]: flags was unused
quote YAML floats

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
     steps:
       - uses: actions/checkout@v4
       - uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php-version }}
           coverage: none
           extensions: intl curl
-      - run: php -v
```
Comment (szepeviktor) [inline .github/workflows/test.yaml:18]: PHP version is included in the output of `shivammathur/setup-php`

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -32,17 +30,17 @@ jobs:
           php-version: '8.1'
           coverage: none
         env:
-          COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPOSER_TOKEN: ${{ github.token }}
```
Comment (szepeviktor) [inline .github/workflows/test.yaml:33]: New name!

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -32,17 +30,17 @@ jobs:
...
           update: true
 
       - name: Install dependencies
-        run: composer install --prefer-dist --no-interaction --no-progress --optimize-autoloader
+        run: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader
```
Comment (szepeviktor) [inline .github/workflows/test.yaml:37]: We do not have a lock file in the repo.

File: .github/workflows/test.yaml [hunk 5]
```diff
@@ -77,11 +75,11 @@ jobs:
...
 
       - name: Run Pint
-        run: ./vendor/bin/pint --test --format=checkstyle | cs2pr
+        run: composer exec -- pint --test --format=checkstyle | cs2pr
```
Comment (szepeviktor) [inline .github/workflows/test.yaml:85]: There is a Composer command to start binaries.

**Comment by spekulatius** [comment]
Thank you @szepeviktor !

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**